.buildkite/bin/update_changelog.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -eu
+
+# $1 CHANGELOG.md file
+# $2 New version
+# $3 Changelog text
+
+if [ -z "$3" ]; 
```

Comment:
Shell check may complain

Reply:
It was fine with the if but did fix some doubled quotes :)